problemsets-mat223/modules/module6.tex

```diff
@@ -165,6 +165,15 @@
 	Let $\ell=\Span\Set*{\mat{1\\2},\mat{-2\\-4}, \mat{1/2\\1}}$. Find
 	two different bases for $\ell$.
 
+    We can see that $\mat{1\\2}$ and $\mat{-2\\-4}$ are linearly {\it{dependent}} because $-2\mat{1\\2} = \mat{-2\\-4}$. However, $\mat{1/2\\1}$ is linearly dependent of $\mat{1\\2}$ and $\mat{-2\\-4}$ as well! And since a basis cannot be formed by a set of linearly dependent vectors, a basis for $\ell$ that contains one of these vectors certainly cannot contain any of the other two. We can use this information to form our two bases; let's call them $\mathcal B_1$ and $\mathcal B_2$.
```

Comment:
Use `\emph{...}` to emphasize text rather than `{\it ...}`.

Break this up into multiple lines rather than having one long line (otherwise I cannot make comments about specific parts).

We should always start by restating the definition, `We are looking for a linearly independent set that spans $\ell$`. Notice that since ...., $\Set*{\mat{1\\2}}$ spans $\ell$. Further ...`